src/Worker/Notifier.php

```diff
@@ -179,7 +179,7 @@ public static function execute($cmd, $target_id)
 
 			// Only deliver threaded replies (comment to a comment) to Diaspora
 			// when the original comment author does support the Diaspora protocol.
-			if ($target_item['parent-uri'] != $target_item['thr-parent']) {
+			if ($thr_parent['author-link'] && $target_item['parent-uri'] != $target_item['thr-parent']) {
```

Comment:
Weren't an `empty(...)` be better for the situation?

Reply:
@MrPetovan do you have an eye on the comment?

Reply:
> Weren't an `!empty(...)` be better for the situation?

No, the field is part of the item query above, so we know the field exists, but it can be null.

> @MrPetovan do you have an eye on the comment?

What do you mean?

Reply:
This was just a poke, so that you don't forget about it.

Reply:
I wrote a reply earlier but I clicked on "Start Review" which put it in the pending state. Your second comment made me realize the mistake even though I didn't understand it.